ZenPacks/zenoss/Import4/bin/imp4_check.sh

```diff
@@ -137,7 +137,7 @@ then
     # extract the perf into the shared staging volume
     status_out "Extracting $perf_tarball"
     info_out "This operation tends to take a long time ..."
-    tar -C "$staging_zenbackup_dir" "$tar_flags" "$perf_tarball" | awk "$awk_cmd" >&2
+    nice -n 5 tar -C "$staging_zenbackup_dir" "$tar_flags" "$perf_tarball" | awk "$awk_cmd" >&2
```

Comment:
What else is happening on the box at this time that you want to make sure doesn't get starved?